extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -142,7 +142,7 @@ class ArrayExtension {
 * http://us2.php.net/manual/en/book.pcre.php
 * see also: http://us2.php.net/manual/en/function.preg-split.php
 */
-    function arraydefine( &$parser, $arrayid, $value='', $delimiter = '/\s*,\s*/', $options = '') {
+    function arraydefine( $parser, $arrayid, $value='', $delimiter = '/\s*,\s*/', $options = '') {
         if (!isset($arrayid))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)